lib/util.js

```diff
@@ -197,6 +198,20 @@ function pad(n) {
   return StringPrototypePadStart(n.toString(), 2, '0');
 }
 
+/**
+ * @param {string} format
+ * @param {string} text
+ * @returns {string}
+ */
+function styleText(format, text) {
+  validateString(text, 'text');
+  const formatCodes = inspect.colors[format];
```

Comment:
does it make sense to do this check on every call? could be done once and cached